src/posix/main.c

```diff
@@ -358,6 +358,11 @@ static const otCliCommand kCommands[] = {
     {"netif", ProcessNetif},
 };
 
+#if OPENTHREAD_CONFIG_CLI_VENDOR_COMMANDS_ENABLE
+extern const otCliCommand gVendorCommands[];
+extern uint8_t            gVendorCommandsLength;
+#endif // OPENTHREAD_CONFIG_CLI_VENDOR_COMMANDS_ENABLE
```

Comment:
can omit when closer than 20 lines
```suggestion
#endif
```